cl/scrapers/tasks.py

```diff
@@ -401,218 +176,37 @@ def extract_recap_pdf(pks, skip_ocr=False, check_if_needed=True):
-    audio_file.tag.publisher_url = u"https://free.law"
-    audio_file.tag.recording_date = audio_obj.docket.date_argued.strftime(
-        "%Y-%m-%d"
-    )
-
-    # Add images to the mp3. If it has a seal, use that for the Front Cover
-    # and use the FLP logo for the Publisher Logo. If it lacks a seal, use the
-    # Publisher logo for both the front cover and the Publisher logo.
-    try:
-        has_seal = seals_data[court.pk]["has_seal"]
-    except AttributeError:
-        # Unknown court in Seal Rookery.
-        has_seal = False
-    except KeyError:
-        # Unknown court altogether (perhaps a test?)
-        has_seal = False
-
-    flp_image_frames = [
-        3,  # "Front Cover". Complete list at eyed3/id3/frames.py
-        14,  # "Publisher logo".
-    ]
-    if has_seal:
-        with open(
-            os.path.join(seals_root, "512", "%s.png" % court.pk), "r"
-        ) as f:
-            audio_file.tag.images.set(
-                3, f.read(), "image/png", u"Seal for %s" % court.short_name
-            )
-        flp_image_frames.remove(3)
-
-    for frame in flp_image_frames:
-        with open(
-            os.path.join(
-                settings.INSTALL_ROOT,
-                "cl",
-                "audio",
-                "static",
-                "png",
-                "producer-300x300.png",
-            ),
-            "r",
-        ) as f:
-            audio_file.tag.images.set(
-                frame,
-                f.read(),
-                "image/png",
-                u"Created for the public domain by Free Law Project",
-            )
-
-    audio_file.tag.save()
-
-
 @app.task
 def process_audio_file(pk):
     """Given the key to an audio file, extract its content and add the related
     meta data to the database.
     """
     af = Audio.objects.get(pk=pk)
-    tmp_path = os.path.join("/tmp", "audio_" + uuid.uuid4().hex + ".mp3")
-    av_path = get_audio_binary()
-    av_command = [
-        av_path,
-        "-i",
-        af.local_path_original_file.path,
-        "-ar",
-        "22050",  # sample rate (audio samples/s) of 22050Hz
-        "-ab",
-        "48k",  # constant bit rate (sample resolution) of 48kbps
-        tmp_path,
-    ]
     try:
-        _ = subprocess.check_output(av_command, stderr=subprocess.STDOUT)
-    except subprocess.CalledProcessError as e:
-        print(
-            "%s failed command: %s\nerror code: %s\noutput: %s\n%s"
-            % (
-                av_path,
-                av_command,
-                e.returncode,
-                e.output,
-                traceback.format_exc(),
-            )
+        response = convert_and_clean_audio(af).json()
+        cf = ContentFile(response["content"])
+        file_name = trunc(best_case_name(af).lower(), 72) + "_cl.mp3"
+        af.file_with_date = af.docket.date_argued
+        af.local_path_mp3.save(file_name, cf, save=False)
+        af.duration = response["duration"]
```

Comment:
Should these lines go after the except clause, before the af.processing_complete line?